File: jvm2/src/Model/Enemy/Buildings/ENRGPlant.kt
package Model.Enemy.Buildings

import Model.Location
import Model.Neutral.Buildings.NRGPlant

class ENRGPlant(location: Location) : NRGPlant() {
    override var loc = location
    override var use = true
    override var owner = 2
    override var icon = "eP"
}

File: dotnet/src/Model/Enemy/Buildings/ENRGPlant.cs
using System;

namespace PlanetCS1 {
    class ENRGPlant : NRGPlant {
        public ENRGPlant(Location location) {
            this.loc = location;
            this.use = true;
            this.owner = 2;
            this.icon = "eP";
        }
    }
}